tutorials/multi-modal/image_classification_tutorial.ipynb

```diff
@@ -0,0 +1,392 @@
+{
```

Comment:
<p>might add a if(px.acvite_session()) session.view at the end</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4441/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>